core/schema/service.go

```
@@ -105,14 +123,87 @@ func (s *Service) Create(ctx context.Context, nsName string, schemaName string,
 		Compatibility: compatibility,
 	}
 	versionID := getIDforSchema(nsName, schemaName, sf.ID)
+	_, prevSchemaData, _ := s.GetLatest(ctx, nsName, schemaName)
```

error is ignored! Please handle it.

---

Maybe report it with log line and a metric and skip the ChangeIdentification, or else the next flows might break

---

we can do metric once we have it implemented. Logged it for now.